### PR TITLE
Add support for renovate bot. Fixes #32

### LINKED ISSUE #32: Support for Renovate bot
Thanks for the great package. 

It's currently missing support for @renovatebot, which is widely used to maintain dependencies. It uses checkboxes in order to trigger re-basing. This means that any PR that is created by @renovatebot will have pending checks when using this package.

Example of renovate PR:
Screenshot 2022-12-16 at 14 52 48

I've submitted a proposed a fix in a PR #31

### PR DESCRIPTION
**Pull request by lukasjuhas**
Hey, thanks for the great package.

We use it at @worksome andd one of the frustration has been using it alongside @renovatebot.

This PR fixes this issue.

Example of renovate PR:
Screenshot 2022-12-16 at 14 52 48

### CODE REVIEW

**Comment by lukasjuhas** [comment]
I considered just comparing the string `[bot]`, but I felt being more specific is better in this case, as people might have bots alongside this action and must check all the boxes.

File: index.js [hunk 1]
```diff
@@ -39,7 +39,13 @@ module.exports = (app) => {
...
     }
 
-    let outstandingTasks = checkOutstandingTasks(pr.body);
+    let outstandingTasks = checkOutstandingTasks(null);
+
+    // if the author is a renovate bot, ignore checks
+    // https://www.mend.io/free-developer-tools/renovate/
+    if (pr.user.login.indexOf('renovate[bot]') === -1) {
+      outstandingTasks = checkOutstandingTasks(pr.body);
+    }
```
Comment (stilliard) [inline index.js]: Looks good but could we flip this statement around so when reading this and adding other bots to be excluded in the future, we default to having the body set but then unset it on exception like so:

```suggestion
    let prBody = pr.body;
    
    // if the author is a renovate bot, ignore checks
    // https://www.mend.io/free-developer-tools/renovate/
    if (pr.user.login.indexOf('renovate[bot]') !== -1) {
      prBody = null;
    }
    
    let outstandingTasks = checkOutstandingTasks(prBody);

```

**Review by stilliard — CHANGES_REQUESTED** [review]
Looks like a useful change. 
Glad you didn't do it for all "[bot]" ones too as I actually have 2 different custom bots I use which each add their own tasks to issues based on the code that's been edited for reminders etc.

File: index.js (hunk 1 above)
Comment (lukasjuhas) [reply index.js]: Great suggestion, thank you @stilliard

**Comment by stilliard** [comment]
Thanks for this @lukasjuhas 
Tested locally and looks like it's working well. 
Releasing it now, should be live in the next few minutes

**Comment by lukasjuhas** [comment]
Thank you so much, @stilliard!

**Pull request merged**